src/batch/src/task/context.rs

```diff
@@ -105,7 +104,7 @@ impl BatchTaskContext for ComputeNodeContext {
     }
 
     fn batch_metrics(&self) -> Option<BatchMetricsWithTaskLabels> {
```

Comment:
May refactor the return type as well.

```suggestion
    fn batch_metrics(&self) -> BatchMetricsWithTaskLabels {
```

Reply:
This one can't be refactored right now, because the frontend node has no `BatchMetricsWithTaskLabels`.